internals/webpack/webpack.base.babel.js

```diff
@@ -138,6 +139,10 @@ module.exports = options => ({
     modules: ['node_modules', 'src'],
     extensions: ['.js', '.jsx', '.react.js'],
     mainFields: ['browser', 'jsnext:main', 'main'],
+    alias: {
+      '@datapunt/asc-ui': path.resolve(__rootdir, 'node_modules/@datapunt/asc-ui/lib'),
+      '@datapunt/asc-assets': path.resolve(__rootdir, 'node_modules/@datapunt/asc-assets/lib'),
+    },
```

Comment:
Why wouldn't we make use of esModules? When I apply them, I'm getting a reduction in production bundle size. Not significant, but still a reduction.

Reply:
I didn't know we don't support IE11 anymore.